---
layout: page
title: Publications (since 2011)
description: Publications
---

Klötzer KA, Abedini A, Balzer MS, Liang X, Levinsohn J, Ha E, Hogan JJ, Quinn G, Bloom RD, Schuller M, Eller K, Zhang NR*, Susztak K*. (2024) Unraveling the Cellular Features of Kidney Disease in a Cross-Species Integrated Single-Cell Kidney Atlas via Pathway-Level Gene Expression Coordination with ScSpectra.  **Submitted**.

Fu Y, Kim H, Adams JI, Grimes SM, Huang S, Lau BT, Sathe A, Hess P, Ji HP, Zhang NR* (2024) Single cell and spatial alternative splicing analysis with long read sequencing.  **Submitted**.

Gier RA, Hueros RA, Rong J, DeMarshall M, Karakasheva TA, Muir AB, Falk GW, Zhang NR, Shaffer SM. (2024) Clonal cell states link Barrett’s esophagus and esophageal adenocarcinoma.  **Submitted**.

1.	Yu W, Biyik-Sit R, Uzun Y, Chen C-H, Thadi A, Sussman JH, Pang M, Wu C-Y, Grossmann LD, Gao P, Wu DW, Yousey A, Zhang M, Turn CS, Zhang Z, Bandyopadhyay S, Huang J, Patel T, Chen C, Martinez D, Surrey LF, Hogarty MD, Bernt K, Zhang NR, Maris JM, Tan K (2025) Longitudinal single-cell multiomic atlas of high-risk neuroblastoma reveals chemotherapy-induced tumor microenvironment rewiring Clonal cell states link Barrett’s esophagus and esophageal adenocarcinoma.  **Nature Genetics, to appear.**

Wu C-Y, Rong J, Sathe A, Hess PR, Lau BT, Grimes SM, Huang S, Ji HP, Zhang NR* (2025). Cancer subclone detection based on DNA copy number in single cell and spatial omic sequencing data.  **Nature Methods, to appear**.

Zhang Z, Mathew D, Lim T, Mason K, Martinez CM, Huang S, Wherry EJ, Susztak K, Minn AJ, Ma Z, Zhang NR* (2025) Signal recovery in single cell batch integration. **Nature Biotechnology**,  published online Nov 26 2024.

Mathew D, Marmarelis ME, Foley C, Bauml M, Ye D, Ghinnagow R, Ngiow SF, Klapholz M, Jun S, Zhang Z, Zorc R, Diehn M, Hwang W-T, Zhang NR, Langer CJ, Wherry EJ, Minn AJ (2024) Clinical and Immunological Responses to Combined JAK Inhibition and PD1 Blockade for Lung Cancer. **Science** 384, 1303-1304.

Mason K, Sathe A, Hess P, Rong J, Wu C-Y, Furth E, Susztak K, Levinsohn J, Ji HP, Zhang NR* (2024) Niche-DE: niche differential gene expression analysis in spatial transcriptomics data identifies context-dependent cell-cell interactions. **Genome Biology** 25, 14.

Chen S, Zhu B, Huang S, Hickey JW, Lin KZ, Snyder M, Greenleaf WJ, Nolan GP, Zhang NR*, Ma Z* (2024) Integration of spatial and single-cell data across modalities with weak linkage.  **Nature Biotechnology** 42, 1096-1106.

Lin K and Zhang NR* (2023) Quantifying common and distinct information in single-cell multimodal data with Tilted Canonical Correlation Analysis.  **Proceedings of the National Academy of Sciences**, 120 (32) e2303647120.

Hickey JW et al. (2023) Organization of the human intestine at single-cell resolution. **Nature** 619 (7970), 572-584.

Sathe A, Mason K, Grimes SM, Zhou Z, Lau BT, Bai X, Su A, Tan X, Lee H, Suarez CJ, Nguyen Q, Poultsides G, Zhang NR, Ji HP (2022) Colorectal cancer metastases in the liver establish immunosuppressive spatial networking between tumor associated SPP1+ macrophages and fibroblasts.  **Clinical Cancer Research** 29, 244.

Jiang Y, Harigaya Y, Zhang Z, Zhang H, Zang C, and Zhang NR* (2022) Nonparametric single-cell multiomic characterization of trio relationships between transcription factors, target genes, and cis-regulatory regions.  **Cell Systems** 13, 737.

Bibby J, Agarwal D, Freiwald T, Kunz N, Merle NS, West EE, Larochelle A, Chinian F, Mukherjee S, Afzali B, Claudia K*, and Zhang NR* (2022).  Systematic Single Cell Pathway Analysis (SCPA) reveals novel pathways engaged during early T cell activation.  **Cell Reports** 41, 111697.

Cucolo L, Chen Q, Qiu J, Yu Y, Klapholz M, Budinich KA, Zhang Z, Shao Y, Brodsky IE, Jordan MS, Gilliland DG , Zhang NR, Shi J and Minn AJ (2022). The interferon-stimulated gene RIPK1 regulates cancer cell intrinsic and extrinsic resistance to immune checkpoint blockade, **Immunity** 55, 671.

Wang J, Zhao Q, Bowden J, Hemani G, Smith GD, Small DS, Zhang NR (2021) Causal inference for heritable phenotypic risk factors using heterogeneous genetic instruments.  **PLoS Genetics** 17(6), e1009575.

Zhao Q, Wang J, Zhen M, Zhang NR, Hennessey S, Small DS and Rader DJ (2021) A Mendelian randomization study of the role of lipoprotein subfractions in coronary artery disease.  **Elife** 10, e58361.  

Navin NE, Rozenblatt-Rosen O, and Zhang NR (2021) New frontiers in single-cell genomics.  **Genome Research**, 31, ix-x.

Wu C-Y, Lau BT, Kim H, Sathe A, Grimes SM, Ji HP, Zhang NR* (2021) Integrative single-cell analysis of allele-specific copy number alterations and chromatin accessibility in cancer.  **Nature Biotechnology**, doi: 10.1038/s41587-021-00911-w 

Mukherjee S, Agarwal D, Zhang NR & Bhattacharya BB (2020) Distribution-Free Multisample Tests Based on Optimal Matchings with Applications to Single Cell Genomics, **Journal of the American Statistical Association**.

Rozenblatt-Rosen et al. (2020) The Human Tumor Atlas Network (HTAN): charting tumor transitions across space and time at single-cell resolution. **Cell** 181, 236.  (Consortia paper as part of HTAN)

Zhou Z, Xu B, Minn A, Zhang NR* (2020) Dendro: genetic heterogeneity profiling by single cell RNA sequencing, **Genome Biology** 21, 10. (R package: DENDRO)

Zhou Z, Ye C, Wang J, Zhang NR* (2020) Surface protein imputation from single cell transcriptomes by deep neural networks, **Nature Communications** 11, Article number: 651

Agarwal D, Wang J, Zhang NR* (2020) Data denoising and post-denoising corrections in single cell RNA sequencing, **Statistical Science** 35 (1), 112-128.

Wang X, Park J, Susztak K, Zhang NR*, Li M* (2019) Bulk Tissue Cell Type Deconvolution with Multi-Subject Single-Cell Expression Reference, **Nature Communications** 10, 380. 

Wang J, Agarwal D, Huang M, Hu G, Zhou Z, Ye C, Zhang NR* (2019) Data denoising with transfer learning in single-cell transcriptomics, **Nature Methods** 16, 875.

Benci JL et al. (2019) Opposing Functions of Interferon Coordinate Adaptive and Innate Immune Responses to Cancer Immune Checkpoint Blockade, **Cell** 178 (4), 933-948. e14.

Nguyen S et al. (2019) Elite control of HIV is associated with distinct functional and transcriptional signatures in lymphoid tissue CD8+ T cells, **Science Translational Medicine** 11(523).

Agarwal D, Zhang NR* (2019) A rank-based semblance kernel on probability spaces, **Science Advances** 5 (12), eaau9630.

Huang M, Wang J, Torre E, Dueck H, Shaffer S, Bonasio R, Murray J, Raj A, Li M, Zhang NR* (2018) SAVER: Gene expression recovery for single cell RNA sequencing, **Nature Methods** 15, 539.

Zhou Z, Wang W, Wang L-S, Zhang NR* (2018) Integrative DNA copy number detection and genotyping from sequencing and array-based platforms, **Bioinformatics** 34, 2349.

Wang X, Jiang Y, Zhang NR, Small D (2018) Sensitivity analysis and power for instrumental variable studies, **Biometrics** doi: 10.1111/biom.12873.

Urrutia E, Chen H, Zhou Z, Zhang NR*, Jiang Y* (2018) Integrative pipeline for profiling DNA copy number and inferring tumor phylogeny, **Bioinformatics** 34, 2126.

Zhang H, Zhang NR, Li M, Reilly MP (2018) First giant steps towards a cell atlas of atherosclerosis, **Circulation Research** 122, 1632.

Wang J, Huang M, Torre E, Dueck H, Shaffer S, Murray J, Raj A, Li M, Zhang NR* (2018) Gene expression distribution deconvolution in single cell RNA sequencing, **Proceedings of the National Academy of Sciences** 115, E6437.

Jiang Y, Nathanson KL, Zhang NR* (2018) CODEX2: full-spectrum copy number variation detection by high-throughput DNA sequencing, **Genome Biology** 19, 202.

Wang X, Chen H, Zhang NR (2017) DNA copy number profiling using single-cell sequencing, **Briefings in Bioinformatics**, bbx004, https://doi.org/10.1093/bib/bbx004.

Jiang Y, Zhang NR*, Li M* (2017) SCALE: modeling allele-specific gene expression by single-cell RNA-sequencing, **Genome Biology** 18, 74.

Chen H, Jiang Y, Maxwell K, Nathanson K, Zhang NR* (2017) Allele-specific copy number estimation by whole exome sequencing, **Annals of Applied Statistics** 11, 1169.

Jia C, Hu Y, Kelly D, Kim J, Li M*, Zhang NR* (2017) Accounting for technical noise in differential expression analysis of single-cell RNA sequencing data, **Nucleic Acids Research**, 45, 10978.

Maxwell KN, Wubbenhorst B, Wenz BM, Sloover DD, Pluta J, Emery L, Barrett A, Kraya AA, Anastopoulos IN, Yu S, Jiang Y, Chen H, Zhang NR, Hackman N, D'Andrea K, Daber R, Morrissette JJ, Mitra N, Feldman M, Domchek SM, Nathanson KL (2017) BRCA locus-speciﬁc loss of heterozygosity in germline BRCA1 and BRCA2 carriers, **Nature Communications** 8, 319.

Xia LC, Bell JM, Wood-Bouwens C, Chen JJ, Zhang NR*, Ji HP* (2017) Single molecule-based discovery of complex genomic rearrangements, **Nucleic Acids Research** 46, e19.

Garman B, Anastopoulos IN, Krepler C, Brafford P, Sproesser K, Jiang Y, Wubbenhorst B,    Amaravadi R, Bennett J, Beqiri M, Elder D, Flaherty KT, Frederick DT, Gangadhar TC, Guarino M, Hoon D, Karakousis G, Liu Q, Mitra N, Petrelli NJ, Schuchter L, Shannan B, Sheilds CL, Wargo J, Wenz B, Wilson MA, Xiao M, Xu W, Xu X, Yin X, Zhang NR, Davies MA, Herlyn M, Nathanson KL (2017) Genetic and genomic characterization of 462 melanoma patient-derived xenografts, tumor biopsies and cell lines, **Cell Reports** 21, 1936.

Wang X, Chen M, Yu X, Pornputtapong N, Chen H, Zhang NR, Powers RS, Krauthammer M (2016) Global copy number profiling of cancer genomes, **Bioinformatics**, 32, 926.

Zhang NR, Yakir B, Xia LC, Siegmund DO (2016) Scan statistics on Poisson random fields with applications in genomics, **Annals of Applied Statistics** 10, 726.

Xia LC, Sakshuwong S, Hopmans ES, Bell JM, Grimes SM, Siegmund DO, Ji HP, Zhang NR* (2016) A genome-wide approach for detecting novel insertion-deletion variants of mid-range size, **Nucleic Acids Research** 44, e126.

Jiang Y, Qiu Y, Minn AJ, Zhang NR* (2016) Assessing intratumor heterogeneity and tracking longitudinal and spatial clonal evolutionary history by next-generation sequencing, **Proceedings of the National Academy of Sciences** 113, E5528.

Chen H, Bell JM, Zavala NA, Ji HP, Zhang NR* (2015) Allele-specific copy number profiling by next-generation DNA sequencing, **Nucleic Acids Research** 43, e23.

Jiang Y, Oldridge DA, Diskin SJ, Zhang NR* (2015) CODEX: a normalization and copy number variation detection method for whole exome sequencing, **Nucleic Acids Research** 43, e39.  

Chen H, Zhang NRǂ (2015) Graph-based change-point detection, **The Annals of Statistics** 43, 139.

Cushing A, Kamali A, Winters M, Hopmans ES, Bell JM, Grimes SM, Li CX, Zhang NR, Moss RB, Holodniy M, Ji H (2015) Emergence of hemagglutinin mutations during the course of influenza infection, **Scientific Reports** 5, 16178.

Peixoto LL, Wimmer ME, Poplawski SG, Tudor JC, Kenworthy CA, Liu S, Mizuno K, Garcia BA, Zhang NR, Giese K, Abel T (2015) Memory acquisition and retrieval impact different epigenetic processes that regulate gene expression, **BMC Genomics** 16, S5.

Yue M, Han X, De Masi L, Zhu C, Ma X, Zhang J, Wu R, Schmieder R, Kaushik RS, Fraser GP, Zhao S, McDermott PF, Weill FX, Mainil JG, Arze C, Fricke WF, Edwards RA, Brisson D, Zhang NR, Rankin SC, Schifferli DM (2015) Allelic variation contributes to bacterial host specificity, **Nature Communications** 6, 8754.

Nadauld LD, Garcia S, Natsoulis G, Bell JM, Miotke L, Hopmans ES, Xu H, Pai RK, Palm C, Regan JF, Chen H, Flaherty P, Ootani A, Zhang NR, Ford JM, Kuo CJ, Ji HP (2014) Metastatic tumor evolution and organoid modeling implicate TGFBR2 as a cancer driver in diffuse gastric cancer, **Genome Biology** 15,428.

Chen H, Zhang NRǂ (2013) Graph-based tests for two-sample comparisons of categorical data,  Statistica Sinica 23, 1479.

Natsoulis G, Zhang NR, Welch K, Bell J, Ji HP (2013) Identification of insertion deletion mutations from deep targeted resequencing,  **Journal of Data Mining in Genomics and Proteomics** 4, 132.

Muralidharan O, Natsoulis G, Bell J, Newburger D, Xu H, Keta I, Ji H, Zhang NR* (2012) A cross-sample statistical model for SNP detection in short-read sequencing data, **Nucleic Acids Research** 40, e5.

Flaherty P, Natsoulis G, Muralidharan O, Winters M, Buenrostro J, Bell J, Brown S, Holodniy M, Zhang N, Ji HP (2012) Ultrasensitive detection of rare mutations using next-generation targeted resequencing, **Nucleic Acids Research** 40, e2.

Shen J, Zhang NR* (2012) Change-point model on nonhomogeneous Poisson processes with application in copy number profiling by next-generation DNA sequencing, **Annals of Applied Statistics** 6, 476.

Muralidharan O, Natsoulis G, Bell J, Ji H, Zhang NR* (2012) Detecting mutations in mixed sample sequencing data using empirical Bayes, **Annals of Applied Statistics** 6, 1047.

Zhang NR, Siegmund DO (2012) Model selection for high dimensional, multi-sequence change-point problems, **Statistica Sinica** 22, 1507.

Sun Y, Zhang NR and Owen A (2012) Multiple hypothesis testing, adjusted for latent variables, with an application to the agemap gene expression data, **Annals of Applied Statistics** 6, 1664.

Chen H, Xing H, Zhang NR* (2011) Estimation of parent specific DNA copy number in tumors using high-density genotyping arrays, **PLoS Computational Biology** 7, e1001060.

Siegmund DO, Yakir B, Zhang NRǂ (2011) Detecting simultaneous variant intervals in aligned sequences, **Annals of Applied Statistics** 5, 645.

Efron B and Zhang NRǂ (2011) False discovery rates and copy number variation, **Biometrika** 98, 251.

Natsoulis G, Bell JM, Xu H, Buenrostro JD, Ordonez H, Grimes S, Newburger D, Jensen M, Zahn JM, Zhang N, Ji HP (2011) A flexible approach for highly multiplexed candidate gene targeted resequencing, **PLoS One** 6, e21088.

Siegmund DO, Zhang NR, Yakir B (2011) False discovery rate for scanning statistics, **Biometrika** 98, 979.
            
<br/><br/> 

**For complete list, see [Google Scholar](https://scholar.google.com/citations?user=6EErockAAAAJ&hl=en).**
